docs/administration-guide/aggregation/ingest-example.md

```diff
@@ -0,0 +1,607 @@
+                    "removed": 5,
+                    "issue": 1
+                }
+            },
+            {
+                "class": "Edge",
+                "group": "Commit",
+                "source": "Dave",
+                "destination": "r1",
+                "directed": true,
+                "properties": {
+                    "added": 6,
+                    "removed": 8,
+                    "issue": 1
+                }
+            },
+            {
+                "class": "Edge",
+                "group": "Commit",
+                "source": "Dave",
+                "destination": "r1",
+                "directed": true,
+                "properties": {
+                    "added": 60,
+                    "removed": 4,
+                    "issue": 2
+                }
+            },
+            {
+                "class": "Edge",
+                "group": "Commit",
+                "source": "Dave",
+                "destination": "r1",
+                "directed": true,
+                "properties": {
+                    "added": 35,
+                    "removed": 10,
+                    "issue": 2
+                }
+            },
+            {
+                "class": "Entity",
+                "group": "Person",
+                "vertex": "Dave"
+            },
+            {
+                "class": "Entity",
+                "group": "Repository",
+                "vertex": "r1"
+            }
+        ]
+    }
+    ```
+
+=== "Result"
+
+    ```json
+    [
+        {
+            "class": "uk.gov.gchq.gaffer.data.element.Entity",
+            "group": "Person",
+            "vertex": "Dave",
+            "properties": {}
+        },
+        {
+            "class": "uk.gov.gchq.gaffer.data.element.Edge",
+            "group": "Commit",
+            "source": "Dave",
+            "destination": "r1",
+            "directed": true,
+            "matchedVertex": "SOURCE",
+            "properties": {
+                "issue": 1,
+                "removed": 13,
+                "added": 26
+            }
+        },
+        {
+            "class": "uk.gov.gchq.gaffer.data.element.Edge",
+            "group": "Commit",
+            "source": "Dave",
+            "destination": "r1",
+            "directed": true,
+            "matchedVertex": "SOURCE",
+            "properties": {
+                "issue": 2,
+                "removed": 14,
+                "added": 95
+            }
+        }
+    ]
+    ```
+
+As you can see we end up with two `Commit` edges relating to each `issue` with
+all other properties aggregated together.
+
+## Expanded Example
+
+The example from the first section is a good demonstration of how aggregation
+works but just having the total number of some properties may not be the most
+useful. To demonstrate a more complex use case we will modify the example to add
+some new properties to the edges so that after aggregation we'll have a graph like:
```

Comment:
```suggestion
works, but just having the total number of some properties may not be the most
useful. To demonstrate a more complex use case we will modify the example to add
some new properties to the edges, so that after aggregation we'll have this graph:
```